scripts/removeOldKibanaIndices.py

```diff
@@ -1,17 +1,16 @@
 import requests
 
-logFile = open('/var/log/probe/KibanaStartup.log', 'w')
+logFile = open('/var/log/probe/KibanaStartup.log', 'w+')
```

Comment:
Sorry, I misled you here. I believe the correct way to open a file to append to it is with mode `a`.
The `w+` is read and write mode.